src/main/java/Time.java

```diff
@@ -0,0 +1,13 @@
+public class Time {
+    int date;
+    int month;
+    int year;
+    int time;
+    public Time(String s) {
+        String[] sa = s.split("/");
+        this.date = Integer.parseInt(sa[0]);
```

Comment:
Can consider adding to check data validity, in case format is incorrect and time input from string is not integer.